cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -36,8 +36,9 @@ function(find_and_configure_raft)
             GIT_TAG        ${PKG_PINNED_TAG}
             SOURCE_SUBDIR  cpp
             OPTIONS
-                "BUILD_TESTS OFF"
                 "RAFT_COMPILE_LIBRARIES OFF"
+                "BUILD_TESTS OFF"
+                "BUILD_BENCH OFF"
```

Comment:
https://github.com/rapidsai/raft/pull/553 gets merged,
and is this still necessary?

Reply:
@cjnolet mentioned that it might be a good idea for clients to explicitly specify these, but maybe he can comment here.

Reply:
I think it's not a bad idea to be explicit about these options. Even if this ends up jsut setting them to their default values in the future, it should protect cugraph from any accidental interruptions that might occur from changes to the default (not that this is planned.. but I personally vote to take the safer route here).